src/parser/xasl_generation.c

```diff
@@ -17336,6 +17340,10 @@ pt_spec_to_xasl_class_oid_list (PARSER_CONTEXT * parser, const PT_NODE * spec, O
 			    {
 			      assert (smclass->stats->heap_num_pages >= 0);
 			      *(t_list + o_num - 1) = smclass->stats->heap_num_pages;
+			      if (smclass->tde_algorithm != TDE_ALGORITHM_NONE)
+				{
+				  *includes_tde_class = 1;
+				}
```

Comment:
else clause is not needed such as "*includes_td_class = 0;"

Reply:
Are you asking? If so, yes, the initial value is 0.

Reply:
that's ok whenever it is called by setting 0